debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -556,7 +556,6 @@ public static DdlParsingMode parse(String value, String defaultValue) {
                                               .withType(Type.PASSWORD)
                                               .withWidth(Width.SHORT)
                                               .withImportance(Importance.HIGH)
-                                              .withValidation(Field::isRequired)
```

Comment:
So can one have an empty password in MySQL?

Reply:
@gunnarmorling Not sure I undertand the question.

Reply:
Sorry; what I meant is this: is it legit to remove this validation for MySQL, or is a non-empty password actually required here and only allowed for Postgres? Does that make sense?

Reply:
@gunnarmorling I envisioned exatcly the same case, you have smart proxy in front of your database when you do not need to proved the credentials. I think we will see more of that when service mesh soltuions like Istio will become more common.

Reply:
Ok then.